util/util.go

```diff
@@ -0,0 +1,26 @@
+package util
```

Comment:
Personally, I would call this package logging instead of util. It handles the logging setup.

Reply:
done!